src/main/java/core/basesyntax/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.bookstore.service.impl;
+
+import core.basesyntax.bookstore.model.Book;
+import core.basesyntax.bookstore.repository.BookRepository;
+import core.basesyntax.bookstore.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+
+    @Autowired

```

Comment:
`@Autowired` is not required for constructor injection, Spring resolves it automatically.